transport/http/src/test/scripts/org/kaazing/gateway/transport/http/security/auth/shouldNotChallengeDueToMismatchingAuthSchemes.rpt

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright 2007-2016, Kaazing Corporation. All rights reserved.
+#
```

Comment:
This script and `shouldChallengeDueToMismatchingAuthSchemes` are the same. Leave just one testing the full flow: no authorization header, then a header with an invalid scheme and finally a valid request.